src/stdlib.js

```diff
@@ -11,6 +11,27 @@ export const lightscriptImports = {
   "bitwiseZeroFillRightShift": "inline",
 };
 
+export const runtimeHelpers = {
```

Comment:
I liked this way of doing helpers a lot, so I tried to adopt it. When I applied it to my local build of `lscdiag` I got some very strange code output in the generated helpers. Took me a while to figure out why -- turns out that minifiers mangle these things and then `fn.toString()` brings in the minified code.

We could just say we don't care about that, but I think live compilers and minifiers are good things generally, whereas this, while neat, could just be replaced with a plain string passed to babel-template. I recommend the latter approach.